### PR TITLE
added new model = default 3-layers model + batch normalization

### PR DESCRIPTION
**Pull request by jngadiub**
Added new model with batch normalization. nb, I haven't tried to prune it.

### CODE REVIEW

**Comment by jngadiub** [comment]
Results of the new 3 layer model + BN layer are very similar to the default 3 layer model:

[ROC.pdf](https://github.com/hls-fpga-machine-learning/keras-training/files/2268124/ROC.pdf)

[confusion_matrix_norm.pdf](https://github.com/hls-fpga-machine-learning/keras-training/files/2268125/confusion_matrix_norm.pdf)
